### PR TITLE
Resolves #2

### LINKED ISSUE #2: Adjustments need to be made to correct branding and URL links here
I am going to correct some branding and URL links here.  This should not be associated with Talos.

### PR DESCRIPTION
**Pull request by finchy**
This Resolves issue #2.  Two additional things I'd like you to add.  A user-agent that identifies (in some way) the SW instance fetching the file, so we can see abusers.

### CODE REVIEW

File: README.md
Comment (kyle-winters) [inline README.md]: This will need to be fixed with new name and no spaces

File: README.md
Comment (kyle-winters) [inline README.md]: please use either "-" or "_" but not both

File: README.md
Comment (kyle-winters) [inline README.md]: please use either "-" or "_" but not both

File: talos_blacklist_importer.py
Comment (kyle-winters) [inline talos_blacklist_importer.py]: remove the underscore please

**Review by kyle-winters — CHANGES_REQUESTED** [review]
Overall looks good. Just some minor changes needed before it can be merged. I will also change the name of the project to SnortBlockListImporter, and the python file to snort_block_list_importer. Please be sure to camelcase SnortBlockListImporter with a capital L

**Comment by finchy** [comment]
Updated.

**Review by kyle-winters — APPROVED** [review]

**Comment by kyle-winters** [comment]
It looks like I dont have write access required to merge the pull request. @TheAlanNix can you please give myself and/or Joel @finchy the necessary permissions please?

**Comment by finchy** [comment]
I think Kyle should have permissions.  I don’t need them.

**Comment by TheAlanNix** [comment]
It looks like an Owner in the CiscoSE organization needs to update the permissions on the repo - @cmlccie can you give @kywinter permissions on this repo?  If memory serves, I think I had sent a list of ~5 repos that he needed to be added to when I left back in May.

**Comment by kyle-winters** [comment]
thanks @TheAlanNix, are you also able to rename the .py file to snort_block_list_importer.py and the project name to SnortBlocklistImporter please?

**Pull request merged**
